src/controller/postUrl.ts

```diff
@@ -0,0 +1,28 @@
+import { NextFunction, Request, Response } from "express";
+import { databaseModel } from "../model";
+import { logger } from "../services";
+import { handleError, sendResponse, STATUS_CODES } from "../utils";
+
+export const postUrl = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    const { originalUrl, customName } = req.body;
+
+    const shortUrl = await databaseModel.create({
+      originalUrl,
+      customName: customName || "",
```

Comment:
Add the base URL to the shortUrl
```suggestion
      customName,
```